README.rst

```diff
@@ -273,6 +273,42 @@ Deleting resources
     cust1.commit() # Actually delete
 
 
+Session history for debugging
+------------------
+
+.. code-block:: python
+
+    # You can use session history to debug http requests.
+    # Session history will be enabled, if you set the log level to DEBUG
+    # or explicitly set a session parameter enable_history_at_log level to a log level
+    # equal or lower than the actual log level.
+
+    # Session history will be enabled by: (Python log level is WARNING by default)
+    s = Session('http://localhost:8080/', schema=models_as_jsonschema,
+                enable_history_at_loglevel='WARNING')
+    # or
+    import logging
+    logging.basicConfig(level='DEBUG')
```

Comment:
Are you sure it's a good idea to lie the library to a logging library this way ? What if I want to use some other logging library ? Or if I want to debug in production but don't want to leak this data to eg. sentry via sentry-sdk breadcrumbs or data logging ? I think this is something that should instead be explicitly set, so eg. enable_session_log: bool instead of enable_history_at_loglevel.

Reply:
Also, it might be a better idea to allow function callback hooks for this. Eg. hook for received data, hook for sent data etc. That would be much more versatile, and also allows for logging.

Reply:
Thanks @katajakasa, I guess you are right! The library user can anyway build whatever logic for binding to log level the way they want, so I will just make it a simple boolean enabled. Adding callback hooks is also a good idea. Let's see what can be done...